static/app/components/replays/replayPlayer.tsx

```diff
@@ -80,27 +74,28 @@ function BasePlayerRoot({
   }, [windowDimensions, videoDimensions]);
 
   return (
-    <Centered ref={windowEl} data-test-id="replay-window">
-      <div ref={viewEl} data-test-id="replay-view" className={className} />
-      {fastForwardSpeed ? (
-        <FastForwardBadge>
-          <FastForwardTooltip title={t('Fast forwarding')}>
-            <IconArrow size="sm" direction="right" />
-            {fastForwardSpeed}x
-          </FastForwardTooltip>
-        </FastForwardBadge>
-      ) : null}
-    </Centered>
+    <Panel isFullscreen={isFullscreen}>
+      <Centered ref={windowEl} data-test-id="replay-window">
+        <div ref={viewEl} data-test-id="replay-view" className={className} />
+        {fastForwardSpeed ? (
+          <FastForwardBadge>
+            <FastForwardTooltip title={t('Fast forwarding')}>
+              <IconArrow size="sm" direction="right" />
+              {fastForwardSpeed}x
+            </FastForwardTooltip>
+          </FastForwardBadge>
+        ) : null}
+      </Centered>
+    </Panel>
   );
 }
 
 const FastForwardBadge = styled('div')`
   position: absolute;
   bottom: 0;
   left: 0;
-  background: ${p => p.theme.purple100};
-  opacity: 0.75;
-  color: ${p => p.theme.textColor};
+  background: ${p => p.theme.gray300};
+  color: ${p => p.theme.white};
```

Comment:
better with the dark theme, per https://storybook.sentry.dev/?path=/story/core-colors--page#grays